submarine-workbench/interpreter/interpreter-engine/pom.xml

```diff
@@ -36,6 +36,18 @@
   <description>Submarine Interpreter Engine</description>
 
   <dependencies>
+    <dependency>
+      <groupId>org.apache.submarine</groupId>
+      <artifactId>commons-utils</artifactId>
+      <version>0.3.0-SNAPSHOT</version>
```

Comment:
Can we change 0.3.0-SNAPSHOT to ${submarine.version}?

Reply:
Done.